ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/electra/block/BlockProcessorElectra.java

```diff
@@ -732,6 +733,16 @@ private Optional<OperationInvalidReason> checkCommittees(
       }
       final IntList committee =
           beaconStateAccessorsElectra.getBeaconCommittee(state, slot, committeeIndex);
+      final int committeeOffset = participantsCount;
+      final long committeeAttesterCount =
+          IntStream.range(0, committee.size())
+              .filter(
+                  committeeParticipantIndex ->
+                      aggregationBits.isSet(committeeOffset + committeeParticipantIndex))
+              .count();
+      if (committeeAttesterCount == 0) {
+        return Optional.of(AttestationInvalidReason.PARTICIPANTS_COUNT_MISMATCH);
+      }
```

Comment:
I think this could be implemented more efficiently as the following:

```java
      final boolean aggregationBitsHasAtLeastOneParticipantFromTheCommittee =
              IntStream.range(0, committee.size())
                      .anyMatch(
                              committeeParticipantIndex ->
                                      aggregationBits.isSet(committeeOffset + committeeParticipantIndex));

      if (!aggregationBitsHasAtLeastOneParticipantFromTheCommittee) {
        return Optional.of(AttestationInvalidReason.PARTICIPANTS_COUNT_MISMATCH);
      }
```